kong/templates/nginx_kong.lua

```diff
@@ -81,7 +81,8 @@ server {
     listen $(entry.listener);
 > end
 
-    error_page 400 404 405 408 411 412 413 414 417 494 /kong_error_handler;
+    error_page 400 404 405 408 411 412 413 414 417 /kong_error_handler;
+    error_page 494 =494                            /kong_error_handler;
```

Comment:
Seems we return `400`. So can remove `=494`?

Reply:
`=494` is essential, or the status code got in error_handler is always 400.